sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_response_handlers.py

```diff
@@ -305,6 +305,8 @@ def lro_get_next_page(lro_status_callback, first_page, continuation_token, show_
     parsed_url = urlparse(continuation_token)
     job_id = parsed_url.path.split("/")[-1]
     query_params = dict(parse_qsl(parsed_url.query.replace("$", "")))
+    if "showStats" in query_params:
+        query_params.pop("showStats")
```

Comment:
preview.4 didn't seem to send "showStats" in the nextLink. It does get sent in preview.5, so we need to pop the camel cased version or else it gets sent to requests and we get an "unexpected keyword argument 'showStats'" TypeError.